da-contracts/contracts/IL1DAValidator.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+struct L1DAValidatorOutput {
+    /// @dev The hash of the uncompressed state diff.
+    bytes32 stateDiffHash;
+    /// @dev The hashes of the blobs on L1. The array is dynamic to account for forward compatibility.
+    /// The length of it must be equal to `maxBlobsSupported`.
+    bytes32[] blobsLinearHashes;
+    /// @dev The commitments to the blobs on L1. The array is dynamic to account for forward compatibility.
+    /// Its length must be equal to the length of blobsLinearHashes.
+    /// @dev If the system supports more blobs than returned, the rest of the array should be filled with zeros.
+    bytes32[] blobsOpeningCommitments;
+}
+
+interface IL1DAValidator {
```

Comment:
Because of compilation reasons. They use the blobHash opcode, which is not supported for hardhat zk compilation.  This was the easiest solution that we saw